### PR TITLE
Websocket connected broadcast & projects

### PR DESCRIPTION
**Pull request by DamianCzajkowski**
this code need to be tested

### CODE REVIEW

File: src/quokka_editor_back/tests/routers/test_connection_manager.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ async def test_websocket_manager_disconnect(websocket: WebSocket):
 
     # When
     await manager.connect(websocket, document_id)
-    manager.disconnect(websocket, document_id)
+    await manager.disconnect(websocket, document_id)
```
Comment (Michal-Suhak) [inline src/quokka_editor_back/tests/routers/test_connection_manager.py:28]: Also

File: src/quokka_editor_back/schema/project.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+from pydantic import BaseModel, Field
+
+
+class ProjectUpdatePayload(BaseModel):
+    title: str | None = Field(None, max_length=250)
```
[the remark this answers is not in the document]
Comment (DamianCzajkowski) [reply src/quokka_editor_back/schema/project.py:5]: now no, I was thinking about this now

File: src/quokka_editor_back/models/document.py [hunk 3]
```diff
@@ -14,7 +15,7 @@ class ShareRole(StrEnum):
 
 class Document(models.Model):
     id = fields.UUIDField(pk=True)
-    title = fields.CharField(max_length=250)
+    title = fields.CharField(max_length=252)
```
[the remark this answers is not in the document]
Comment (Michal-Suhak) [reply src/quokka_editor_back/models/document.py:18]: ok

File: src/quokka_editor_back/routers/websockets.py [hunk 4]
```diff
@@ -105,7 +106,7 @@ async def websocket_endpoint(
     except WebSocketDisconnect:
         listen_task.cancel()
     finally:
-        manager.disconnect(websocket, document_id)
+        await manager.disconnect(websocket, document_id)
```
[the remark this answers is not in the document]
Comment (DamianCzajkowski) [reply src/quokka_editor_back/routers/websockets.py:109]: no, we set this @sync_to_async decorator to this func

**Review by Michal-Suhak — APPROVED** [review]
Good job

**Pull request merged**